src/main/java/slimeknights/tconstruct/tools/stats/HandleMaterialStats.java

```diff
@@ -25,40 +24,51 @@
 public class HandleMaterialStats extends BaseMaterialStats {
 
   public static final MaterialStatsId ID = new MaterialStatsId(Util.getResource("handle"));
-  public static final HandleMaterialStats DEFAULT = new HandleMaterialStats(1f, 0);
+  public static final HandleMaterialStats DEFAULT = new HandleMaterialStats(0, 1f, 1f, 1f);
 
-  public final static String MULTIPLIER_LOCALIZATION = "stat.handle.modifier.name";
-  public final static String DURABILITY_LOCALIZATION = "stat.handle.durability.name";
+  public final static String MULTIPLIER_LOCALIZATION = "stat.handle.durability.multiplier.name";
+  public final static String DURABILITY_LOCALIZATION = "stat.handle.durability.base.name";
+  public final static String ATTACK_SPEED_LOCALIZATION = "stat.handle.speed.attack.name";
+  public final static String MINING_SPEED_LOCALIZATION = "stat.handle.speed.mining.name";
 
-  public final static String MULTIPLIER_DESCRIPTION_LOCALIZATION = "stat.handle.modifier.description";
-  public final static String DURABILITY_DESCRIPTION_LOCALIZATION = "stat.handle.durability.description";
+  public final static String MULTIPLIER_DESCRIPTION_LOCALIZATION = "stat.handle.durability.multiplier.description";
+  public final static String DURABILITY_DESCRIPTION_LOCALIZATION = "stat.handle.durability.base.description";
+  public final static String ATTACK_SPEED_DESCRIPTION_LOCALIZATION = "stat.handle.speed.attack.description";
+  public final static String MINING_SPEED_DESCRIPTION_LOCALIZATION = "stat.handle.speed.mining.description";
 
   public final static Color DURABILITY_COLOR = HeadMaterialStats.DURABILITY_COLOR;
   public final static Color MODIFIER_COLOR = Color.fromInt(0xFFB9B95A);
 
-  private float modifier;
   private int durability;
```

Comment:
Can we just ditch the flat durability? Do only modifiers here. Even if we never use it internally, I'd rather not require that complexity in stat calculations.